app/src/main/java/net/squanchy/imageloader/ImageLoaderModule.java

```diff
@@ -0,0 +1,21 @@
+package net.squanchy.imageloader;
+
+import android.content.Context;
+
+import dagger.Module;
+import dagger.Provides;
+
+@Module
+class ImageLoaderModule {
+
+    private final Context context;
+
+    ImageLoaderModule(Context context) {
+        this.context = context;
+    }
+
+    @Provides
+    ImageLoader imageLoader() {
```

Comment:
here you would have `imageLoader(Glide glide)` and a `@Provides Glide glide() { return Glide.with(context); }`